bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfile.java

```diff
@@ -233,7 +233,9 @@ public void initFromThingType(String name) {
                 || thingType.equals(THING_TYPE_SHELLYBLUBUTTON_STR);
         isTRV = thingType.equals(THING_TYPE_SHELLYTRV_STR);
         isWall = thingType.equals(THING_TYPE_SHELLYPLUSWALLDISPLAY_STR);
-        is3EM = thingType.equals(THING_TYPE_SHELLY3EM_STR) || thingType.startsWith(THING_TYPE_SHELLYPRO3EM_STR);
+        is3EM = thingType.equals(THING_TYPE_SHELLY3EM_STR) || thingType.startsWith(THING_TYPE_SHELLYPRO3EM_STR)
+                || thingType.startsWith(THING_TYPE_SHELLYPLUSEM_STR)
+                || thingType.startsWith(THING_TYPE_SHELLYPLUS3EM63_STR);
```

Comment:
When rebasing and resolving conflicts you simply need to compare full `ThingTypeUID` rather than the thing type id string:
```java
THING_TYPE_SHELLYPLUSEM.equals(thingTypeUID) || THING_TYPE_SHELLYPLUS3EM63.equals(thingTypeUID)
```

I don't know why you used `startsWith` here, I don't think that's necessary or desirable.

As a side note, I would like to do some grouping here as well, but didn't want to go that far with #18923. This can be done after 5.0 without touching `ShellyThingCreator` where all the conflicts happen.